Packs/DefaultPlaybook/Playbooks/Default_8_2.yml

```diff
@@ -979,6 +1042,76 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+    scriptarguments:
```

Comment:
Question - if those are the default inputs and their values, if you delete this section - it will be taken directly from the sub playbook values (means easier maintenance). WDYT?